manticore/platforms/evm.py

```diff
@@ -2381,3 +2382,18 @@ def compare_buffers(a, b):
             
         self.current._push(value)
         self.current.pc += self.current.instruction.size
+
+    def generate_workspace_files(self):
+        ret = {}
+        for i, tx in enumerate(self.transactions):
+            name = 'tx.{}'.format(i)
+            data = {
+                'to': tx.address,
+                'from': tx.caller,
+                'data': solver.get_value(self.constraints, tx.data).encode('hex'),
+                'value': tx.value if not issymbolic(tx.value) else '{symbolic!}'
```

Comment:
For `data` it provides a concrete solution but for `value` `{symbolic!}`.
Maybe provide a concrete solution for value too ?